### PR TITLE
OSDOCS-3709: Adding Important block for FC storage on arm

### PR DESCRIPTION
**Pull request by kelbrown20**
For Versions: 4.11+
Jira Story: [OSDOCS-3709](https://issues.redhat.com//browse/OSDOCS-3709)

Preview:
[Persistent storage using Fibre Channel ](https://kelbrown20.github.io/OCP-on-arm-doc-previews-4.11/OSDOCS-3709-storage-options-for-arm/storage/persistent_storage/persistent-storage-fibre.html)

### CODE REVIEW

**Comment by kelbrown20** [comment]
@lwan-wanglin Hi Lin! Could you also take a look at this one from the QE side? Thank you!

File: storage/persistent_storage/persistent-storage-fibre.adoc [hunk 1]
```diff
@@ -24,6 +24,11 @@ High availability of storage in the infrastructure is left to the underlying
 storage provider.
 ====
 
+[IMPORTANT]
+====
+Persistent storage using Fiber Channel is not supported on ARM architecture based infrastructures.
```
Comment (lwan-wanglin) [inline storage/persistent_storage/persistent-storage-fibre.adoc:29]: typo issue? should it be `Fibre Channel`?

**Comment by lwan-wanglin** [comment]
/lgtm
Thanks @kelbrown20

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**